### PR TITLE
Add calico version support into download calico

### PR DESCRIPTION
**Pull request by bplasmeijer**
The bug fix will resolve bug #227.

### CODE REVIEW

File: experiments/image-builder/overlays/packer-windows.json [hunk 1]
```diff
@@ -87,6 +87,7 @@
       "vars_inline": {
         "OS": "{{user `distro_name` | lower}}",
         "PROVIDER": "virtualbox",
+        "calico_version": "{{user `calico_version`}}",
```
Comment (knabben) [inline experiments/image-builder/overlays/packer-windows.json:90]: we dont need to pass down here, it's not used when building the host

File: variables.yaml [hunk 2]
```diff
@@ -21,3 +21,4 @@ windows_cpus: 4
 ## CHOOSE YOUR CNI ($)
 # cni: "calico" || "antrea"
 cni: "calico"
+calico_version: "3.25"
```
Comment (knabben) [inline variables.yaml:24]: can you move up, below containerd_version?

File: Vagrantfile [hunk 3]
```diff
@@ -79,7 +80,7 @@ Vagrant.configure(2) do |config|
 
     if not File.file?(".lock/joined") then
      # Update contaienrd
```
Comment (knabben) [inline Vagrantfile:82]: can you fix the typo `containerd`.

**Comment by knabben** [comment]
a few nits. besides that If we are adding this variable, can we set this variable in the linux configuration as well..
https://github.com/kubernetes-sigs/sig-windows-dev-tools/blob/master/sync/linux/calico-0.sh#L27

**Comment by bplasmeijer** [comment]
> a few nits. besides that If we are adding this variable, can we set this variable in the linux configuration as well.. https://github.com/kubernetes-sigs/sig-windows-dev-tools/blob/master/sync/linux/calico-0.sh#L27

I will take this one tomorrow.

**Comment by bplasmeijer** [comment]
> > a few nits. besides that If we are adding this variable, can we set this variable in the linux configuration as well.. https://github.com/kubernetes-sigs/sig-windows-dev-tools/blob/master/sync/linux/calico-0.sh#L27
> 
> I will take this one tomorrow.

fixed

**Comment by aravindhp** [comment]
tide/merge-method-squash

**Comment by knabben** [comment]
/lgtm

@bplasmeijer let me know if you are still testing or it's good to go?

**Comment by bplasmeijer** [comment]
I can confirm fresh boot, vagrant up --provision, and fully tested, @knabben 
```
controlplane   Ready    control-plane   21m
winw1          Ready    <none>          2m57s
```

**Comment by knabben** [comment]
great job!

/approve

**Pull request merged**